ConfigurationSystem/scripts/dirac-admin-add-resources.py

```diff
@@ -313,10 +313,10 @@ def checkUnusedSEs():
         changeSet.add( ( seSection, 'VO', ','.join( seVOs ) ) )
         accessSection = cfgPath( seSection, 'AccessProtocol.1' )
         changeSet.add( ( accessSection, 'Protocol', 'srm' ) )
-        changeSet.add( ( accessSection, 'ProtocolName', 'SRM2' ) )
+        changeSet.add( ( accessSection, 'PluginName', 'SRM2' ) )
```

Comment:
This is a new non-contradictory option name to be used now in the CS. The backward compatibility with the old ambiguous  ProtocolName is preserved.